lib/2_uppercases_all_letters.rb

```diff
@@ -5,5 +5,6 @@
 # Run `bin/check` from the command line to execute the automated tests.
 
 def uppercases_all_letters(str)
-  # Write your code here.
+    answer = str.upcase
```

Comment:
Nice! How might you condense the work in this method to one line? Thinking about how the variable `answer` is being used may help.